Update/kage_012.txt

```diff
@@ -1702,28 +1702,28 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 37, "ps3/s08/37/320300398", 256, TRUE);
 	OutputLine(NULL, "「ああわからないさ！",
-			NULL, "\"Aah, I don't know anything!", Line_WaitForInput);
+			NULL, "\"Yeah, I don't know you!", Line_WaitForInput);
 	ModPlayVoiceLS(4, 37, "ps3/s08/37/320300399", 256, TRUE);
 	OutputLine(NULL, "　わかりたくもないね！",
-			NULL, " You don't want me to understand! ", GetGlobalFlag(GLinemodeSp));
+			NULL, " I don't want to know! ", GetGlobalFlag(GLinemodeSp));
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 37, "ps3/s08/37/320300400", 256, TRUE);
 	OutputLine(NULL, "　あんたの田舎じゃ、あうんの呼吸で思いが通じ合えるのかもしれないけどね、あいにく私は超能力者でも心理学者でもないんだ！」",
-			NULL, "You have a completely different upbringing and a completely different way of thinking, so it must be difficult for you to communicate. Unfortunately, I'm not a psychologist, and I'm not psychic!\"", Line_Normal);
+			NULL, "Maybe out in that country of yours, you can tell what everyone else is thinking through harmonizing, but unfortunately for you, I'm not an esper or a psychologist!\"", Line_Normal);
 	ClearMessage();
 
 //千紗登rvS08/37/320300401.「言葉にしてくれなきゃ伝わらないし、あんたの気持ちもさっぱりわかりゃしないんだよっ！！」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 37, "ps3/s08/37/320300401", 256, TRUE);
 	OutputLine(NULL, "「言葉にしてくれなきゃ伝わらないし、あんたの気持ちもさっぱりわかりゃしないんだよっ！！」",
-			NULL, "\"If you can't talk to me and tell me how you feel, I'll never understand!!\"", Line_Normal);
+			NULL, "\"Unless you tell me directly how you feel, I'll never understand!!\"", Line_Normal);
 	ClearMessage();
 
 //千紗登rvS08/37/320300402.「そんなふうに甘え根性だから、あんたはいつまでたっても田舎者で、この町じゃよそ者扱いなんだ！！」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 37, "ps3/s08/37/320300402", 256, TRUE);
 	OutputLine(NULL, "「そんなふうに甘え根性だから、あんたはいつまでたっても田舎者で、この町じゃよそ者扱いなんだ！！」",
-			NULL, "\"But you can't do that, because you're still stuck in your reclusive country ways, and you're not willing to open up to anyone in this town!!\"", Line_Normal);
+			NULL, "\"It's because you always depend on others like that that you'll always be treated like a country bumpkin and an outsider in this town no matter how much time passes!!\"", Line_Normal);
```

Comment:
```suggestion
			NULL, "\"It's because you always depend on others like that!! That's why you'll always be treated like a country bumpkin and an outsider in this town no matter how much time passes!!\"", Line_Normal);
```